src/scaling.jl

```diff
@@ -0,0 +1,192 @@
+"""
+    Scaling <: Transform
+
+Linearly scale the data as `ax + b`, according to some statistics `a` and `b`.
+"""
+abstract type Scaling <: Transform end
+
+"""
+    IdentityScaling
+
+Represents the no-op scaling which simply returns the data it is applied on.
+"""
+struct IdentityScaling <: Scaling end
+# Convenience method
+IdentityScaling(args...; kwargs...) = IdentityScaling()
+
+apply!(A::AbstractArray{T}, scaling::IdentityScaling; kwargs...) where T <: Real = A
+
+"""
+    MeanStdScaling(mean, std) <: Scaling
+
+Linearly scale the data by a statistical `mean` and standard deviation `std`.
+The first `mean` and `std` calculated on some data will remain fixed.
+"""
+struct MeanStdScaling <: Scaling
+    # Dict keeps track of slices to protect against changes e.g. Table column order
+    # Is this going too far to generalise between Arrays and Tables? Should we use
+    # separate/parameterised structs?
+    mean::Dict{Symbol, AbstractFloat}
+    std::Dict{Symbol, AbstractFloat}
+
+    """
+        MeanStdScaling() <: Scaling
+
+    Construct a `MeanStdScaling` transform which will populate its `mean` and `std`
+    parameters from the first data it is applied to.
+    """
+    MeanStdScaling() = new(Dict(), Dict())
+end
+
+"""
+    MeanStdScaling(data; kwargs...) <: Scaling
+
+Construct a [`MeanStdScaling`](@ref) using the mean and standard deviation of the data.
+
+# Keyword arguments
+*
+"""
+function MeanStdScaling(data; kwargs...)
+    scaling = MeanStdScaling()
+    populate_stats!(scaling, data; kwargs...)
+    return scaling
+end
```

Comment:
I know this method existed in FeatureEngineering but note this is already covered by making `Transform`s themselves callable types. 

So possibly you are overloading this unnecessarily and in a wrong way...